### PR TITLE
Rebrand to WebThings Gateway.

### PR DESCRIPTION
**Pull request by mrstegeman**

### CODE REVIEW

**Comment by benfrancis** [comment]
@mrstegeman Thanks for doing this. I will review following branding discussions. Will also need to replace wordmark.

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "things-gateway",
```
[the remark this answers is not in the document]
Comment (mrstegeman) [reply package.json]: Given that we don't publish this to npm, it shouldn't really matter.

File: src/mdns-server.js [hunk 2]
```diff
@@ -163,8 +163,8 @@ class DNSserviceDiscovery {
    *
    * @param {object} newProfile For example:
    *   newProfile = {
-   *     name: "Mozilla IoT Gateway",
-   *     host: "mozillaGateway",     // Local DNS name
+   *     name: "WebThings Gateway",
+   *     host: "gateway",     // Local DNS name
```
[the remark this answers is not in the document]
Comment (mrstegeman) [reply src/mdns-server.js]: This is just a duplication of what's in the config file.

[the remark this answers is not in the document]
Comment (mrstegeman) [reply src/mdns-server.js]: If you want to switch from `gateway.local` to `webthing.local`, can we do that separately? That'll have to touch quite a few things.

[the remark this answers is not in the document]
Comment (benfrancis) [reply src/mdns-server.js]: Sure, not a priority.

**Pull request merged**